src/applications/yellow-ribbon/containers/SearchForm/index.jsx

```diff
@@ -132,7 +132,7 @@ export class SearchForm extends Component {
 
         {/* State Field */}
         <label htmlFor="yr-search-state" className="vads-u-margin-top--3">
-          State or Territory
+          State or territory
```

Comment:
Interesting approach. Any risks we should be aware of with this change? A potential new on-call policy?

Reply:
LOL You should write a book on how to best troll! 🤣